Logger.Tests/Logger.Tests.csproj

```
@@ -1,9 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net7.0</TargetFramework>
+		<TargetFramework>net8.0</TargetFramework>
 		<IsPackable>false</IsPackable>
-		<LangVersion>11.0</LangVersion>
+		<LangVersion>12.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <EnableNETAnalyzers>True</EnableNETAnalyzers>
+    <Nullability>True</Nullability>
+    <CodeAnalysisTreatWarningAsErrors>True</CodeAnalysisTreatWarningAsErrors>
+    <EnforceCodeStyleInBuild>True</EnforceCodeStyleInBuild>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
```

```suggestion
		<LangVersion>12.0</LangVersion>
		<Nullable>enable</Nullable>
		<EnableNETAnalyzers>true</EnableNETAnalyzers>
		<Nullability>true</Nullability>
		<CodeAnalysisTreatWarningAsErrors>true</CodeAnalysisTreatWarningAsErrors>
		<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
casing was wrong. And spacing should be consistent.